ladybug/windrose.py

```diff
@@ -188,19 +232,37 @@ def legend_parameters(self):
         Default segment count is 10.
         """
         if self._legend_parameters is None:
-            self._legend_parameters = LegendParameters(segment_count=10)
+            self._legend_parameters = self.container.legend_parameters
         return self._legend_parameters
 
     @legend_parameters.setter
     def legend_parameters(self, legend_parameters):
         assert isinstance(legend_parameters, LegendParameters), 'legend_parameters' \
             ' must be a LegendParameters. Got {}.'.format(type(legend_parameters))
-
-        # This will mutate the legend_parameters in the windrose Legend object.
         self._compass = None
         self._container = None
         self._legend_parameters = legend_parameters
 
+    @property
+    def north(self):
+        """North orientation for windrose by degrees in counterclockwise orienation.
+
+        This must be a number between -360 and 360. 0 is North, 90 is West and 270
+        is East.
+        """
+        if self._north is None:
+            self._north = self.DEFAULT_NORTH
+        return self._north
+
+    @north.setter
+    def north(self, north):
+        assert isinstance(north, float), 'north must be a float representing degrees.' \
+            ' Got {}.'.format(north)
+
+        self._compass = None
+        self._container = None
+        self._north = north % 360.0
```

Comment:
Why do you divide by 360 here? Is the `_north` not in degrees?

Reply:
Yes this is in degrees. But it's not a slash, it's a modulo sign (%). It's just there to reset values > 360 back to 0.

Reply:
Ah, that's actually very elegant. Sorry that I forgot the modulo is the remainder of the division and not the result of the division itself.